### PR TITLE
fix: typo spelling grammar

### PR DESCRIPTION
**Pull request by slowy07**
# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/main/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running `generate.ps1`. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version.
	
	**Note: We have recently updated the PSH module called by `generate.ps1` to emit additional data. This would help reduce/eliminate the Code Verification check error. Please run following command**:

	    `dotnet msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules`

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

**Review by jsquire — APPROVED** [review]
Hi @slowy07.   Thank you for your contribution and for helping to improve the Azure SDK experience. 

I've taken a look at the Event Hubs changes and they look good to me.   We'll want to give the other library owners a chance to review changes in their area, so we'll need to wait a bit before merging this in.

**Comment by slowy07** [comment]
thanks sir @jsquire !!

**Review by LarryOsterman — APPROVED** [review]
@slowy07 The attestation changes look fine, thank you for catching and fixing my spelling mistakes.

**Review by heaths — APPROVED** [review]
Key Vault LGTM

**Comment by slowy07** [comment]
thank your sir @LarryOsterman @heaths !!

**Comment by jsquire** [comment]
@allenjzhang, @m-nash:  Would either of you be so kind as to offer your thoughts and, potentially, assist with resolving the management pipeline build error?

**Review by christothes — APPROVED** [review]
Identity looks good.

**Review by Mohit-Chakraborty — APPROVED** [review]

**Comment by yungezz** [comment]
hi @HarveyLink could you help to look at the mgmt CI task failure? thanks

File: sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Generated/Models/KafkaRestProperties.cs
Comment (HarveyLink) [inline sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Generated/Models/KafkaRestProperties.cs:38]: If the file is under Generated folder, we can't simply change it like that. A swagger change is required and re-run the generate.ps1 to generate the sdk.

File: sdk/dns/Microsoft.Azure.Management.Dns/tests/Helpers/ResourceGroupHelpers.cs
Comment (HarveyLink) [inline sdk/dns/Microsoft.Azure.Management.Dns/tests/Helpers/ResourceGroupHelpers.cs:59]: DNS is a left over question needs to be solved. We should update the readme in swagger repo, add .NET block and trying to regenerate it. For this PR, If possible, I suggest we can exclude this change for now.

Comment (jsquire) [reply sdk/dns/Microsoft.Azure.Management.Dns/tests/Helpers/ResourceGroupHelpers.cs:59]: I agree;  the best path forward would be to revert this change to unblock the rest of the PR.

Comment (slowy07) [reply sdk/dns/Microsoft.Azure.Management.Dns/tests/Helpers/ResourceGroupHelpers.cs:59]: thanks sir @HarveyLink  for review !, that means the best way is for me to return it as it was right?

Comment (jsquire) [reply sdk/dns/Microsoft.Azure.Management.Dns/tests/Helpers/ResourceGroupHelpers.cs:59]: That's correct, @slowy07.  If you undo your changes to the following, that should pass CI and unblock getting things merged:

- `sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Generated/Models/KafkaRestProperties.cs`
- `sdk/dns/Microsoft.Azure.Management.Dns/tests/Helpers/ResourceGroupHelpers.cs`

Comment (slowy07) [reply sdk/dns/Microsoft.Azure.Management.Dns/tests/Helpers/ResourceGroupHelpers.cs:59]: sorry sir @jsquire, how can undo the commit ??

**Comment by slowy07** [comment]
revert specific file on commit [cd9cf62](https://github.com/Azure/azure-sdk-for-net/pull/23131/commits/cd9cf62b7d858972b8ed959aa8bfc562f8cd0210) and [2704071](https://github.com/Azure/azure-sdk-for-net/pull/23131/commits/27040714ff69281883312f842af3de7df3e6c3f1)

**Comment by HarveyLink** [comment]
LGTM

**Review by HarveyLink — APPROVED** [review]

**Comment by jsquire** [comment]
CI and sign-offs look good;  merging this in.    @slowy07:  Thank you, again, for your contribution.

**Comment by slowy07** [comment]
thank you sir @jsquire for teach me more !

**Pull request merged**
